Linters/PSScriptAnalyzerSettings.psd1

```diff
@@ -0,0 +1,12 @@
+@{
+    Severity=@("Error", "Warning")
+    ExcludeRules=@(
+        "PSAvoidUsingWriteHost",        # This will be removed in favor of Write-Log in the future.
+        "PSAvoidUsingInvokeExpression", # We don't have an alternative for this yet.
+        "PSUseShouldProcessForStateChangingFunctions",      # We don't care about altering system state.
+        "PSAvoidUsingConvertToSecureStringWithPlainText",   # We have creds mostly to testbed
```

Comment:
it's a little weird that you disable it globally but disable PSAvoidUsingPlainTextForPassword locally